test/functional/unified-spec-runner/operations.ts

```diff
@@ -274,7 +274,9 @@ operations.set('insertOne', async ({ entities, operation }) => {
     session
   } as InsertOneOptions;
 
-  return collection.insertOne(operation.arguments.document, options);
+  const result = await collection.insertOne(operation.arguments.document, options);
+  result.insertedCount = 1;
+  return result;
```

Comment:
Not sure how you feel about this, but `insertedCount` does not exist on `insertOne` return values and is expected by the spec tests

Reply:
That actually sounds like a spec oversight, I think we'll need to double check with the leads, because that expectation does not seem to match the spec definition @nbbeeken can you take point on this next week since I'll be out?

Reply:
```typescript
  result.insertedCount = 1; // TODO(NODE-3385): Remove insertedCount patch
```
I made us NODE-3385 to keep track of this, but we can keep the patch to proceed with the PR

Reply:
This should be addressed by pulling in the new spec changes :+1: